editor/src/core/workers/bundler-bridge.ts

```diff
@@ -21,23 +21,23 @@ export interface BundlerContext {
   }
 }
 
-type BundlerSchema = {
+interface BundlerSchema extends StateSchema {
   states: {
     worker: {
       states: {
-        uninitialized: {}
-        initializing: {}
-        initagain: {}
-        idle: {}
-        processing: {}
-        error: {}
+        uninitialized: StateSchema
```

Comment:
did you look up the xstate docs for this? does the base type StateSchema represent an empty schema?

Reply:
This is `StateSchema`:
```
export interface StateSchema {
    meta?: any;
    states?: Record<string | number, StateSchema>;
}
```

Assuming this is fine as an empty value is okay.